src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -53,10 +53,16 @@ def create_ramp_fit_class(model, dqflags=None, suppress_one_group=False):
     """
     ramp_data = ramp_fit_class.RampData()
 
+    if not hasattr(model, 'average_dark_current'):
```

Comment:
Nice! This (and your other changes) appear to have appeased `romancal`. Let me know when you're ready for another regtest run.